src/Client.php

```diff
@@ -21,6 +24,10 @@
 
 final class Client
 {
+    use EventEmitterTrait;
```

Comment:
I added this small dependency `evenement/evenement` because it's a very convenient but lean way to subscribe to events